docs/userguide.rst

```diff
@@ -221,3 +221,31 @@ were deployed during ``init``, you can run:
 .. code-block:: console
 
    $ reana-cluster down
+
+Delete interactive sessions
+---------------------------
+
+Interactive sessions that were not closed after work have been finished leave
+active Kubernetes objects (pod, service and ingress) running in the Reana cluster.
+These objects might need to be deleted from time to time. Please note that
+interactive session pod and service are linked to the ingress object. If ingress
```

Comment:
What do you think about highlighting Kubernetes objects like `pod`, `service` etc...?